cdist/conf/type/__install_config/gencode-local

```diff
@@ -24,16 +24,16 @@ remote_copy="$__type/files/remote/copy"
 
 cdist_args=""
 case "$__cdist_loglevel" in
```

Comment:
If cdist itself honors an existing __cdist_loglevel than this is not required anymore.
Nested cdist invocations would automatically inherit this from the parents environment.

Reply:
But with current implementation only emulator honors this.
cdist itself, when started, does not.
I just replaced __debug with __cdist_loglevel.